sycl/source/handler.cpp

```diff
@@ -106,6 +111,17 @@ event handler::finalize() {
     return MLastEvent;
   MIsFinalized = true;
 
+  // If we have a subgraph node that means that a subgraph was recorded as
+  // part of this queue submission, so we skip adding a new node here since
+  // they have already been added, and return the event associated with the
+  // subgraph node.
+  if (MQueue && MQueue->getCommandGraph() && MSubgraphNode) {
+    {
+      return detail::createSyclObjFromImpl<event>(
+          MQueue->getCommandGraph()->get_event_for_node(MSubgraphNode));
+    }
```

Comment:
```suggestion
    return detail::createSyclObjFromImpl<event>(
        MQueue->getCommandGraph()->get_event_for_node(MSubgraphNode));
```
Do we really need extra braces here?